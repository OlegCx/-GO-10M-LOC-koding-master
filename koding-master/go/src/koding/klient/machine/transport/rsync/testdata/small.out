receiving incremental file list
cd+++++++++ test/
>f+++++++++ test/file.01
              0   0%    0.00kB/s    0:00:00            6,144 100%    5.86MB/s    0:00:00 (xfr#1, to-chk=29/31)
>f+++++++++ test/file.02
              0   0%    0.00kB/s    0:00:00            7,168 100%    6.84MB/s    0:00:00 (xfr#2, to-chk=28/31)
>f+++++++++ test/file.03
              0   0%    0.00kB/s    0:00:00           10,240 100%    1.40MB/s    0:00:00 (xfr#3, to-chk=27/31)
>f+++++++++ test/file.04
              0   0%    0.00kB/s    0:00:00            6,144 100%  857.14kB/s    0:00:00 (xfr#4, to-chk=26/31)
>f+++++++++ test/file.05
              0   0%    0.00kB/s    0:00:00            6,144 100%   93.75kB/s    0:00:00 (xfr#5, to-chk=25/31)
>f+++++++++ test/file.06
              0   0%    0.00kB/s    0:00:00            8,192 100%  125.00kB/s    0:00:00 (xfr#6, to-chk=24/31)
>f+++++++++ test/file.07
              0   0%    0.00kB/s    0:00:00            2,048 100%   31.25kB/s    0:00:00 (xfr#7, to-chk=23/31)
>f+++++++++ test/file.08
              0   0%    0.00kB/s    0:00:00            5,120 100%   67.57kB/s    0:00:00 (xfr#8, to-chk=22/31)
>f+++++++++ test/file.09
              0   0%    0.00kB/s    0:00:00            2,048 100%   27.03kB/s    0:00:00 (xfr#9, to-chk=21/31)
>f+++++++++ test/file.10
              0   0%    0.00kB/s    0:00:00            7,168 100%   94.59kB/s    0:00:00 (xfr#10, to-chk=20/31)
>f+++++++++ test/file.11
              0   0%    0.00kB/s    0:00:00            7,168 100%   88.61kB/s    0:00:00 (xfr#11, to-chk=19/31)
>f+++++++++ test/file.12
              0   0%    0.00kB/s    0:00:00            5,120 100%   63.29kB/s    0:00:00 (xfr#12, to-chk=18/31)
>f+++++++++ test/file.13
              0   0%    0.00kB/s    0:00:00            3,072 100%   37.97kB/s    0:00:00 (xfr#13, to-chk=17/31)
>f+++++++++ test/file.14
              0   0%    0.00kB/s    0:00:00            9,216 100%   78.26kB/s    0:00:00 (xfr#14, to-chk=16/31)
>f+++++++++ test/file.15
              0   0%    0.00kB/s    0:00:00            3,072 100%   26.09kB/s    0:00:00 (xfr#15, to-chk=15/31)
>f+++++++++ test/file.16
              0   0%    0.00kB/s    0:00:00            7,168 100%   60.87kB/s    0:00:00 (xfr#16, to-chk=14/31)
>f+++++++++ test/file.17
              0   0%    0.00kB/s    0:00:00            6,144 100%   47.24kB/s    0:00:00 (xfr#17, to-chk=13/31)
>f+++++++++ test/file.18
              0   0%    0.00kB/s    0:00:00            2,048 100%   15.75kB/s    0:00:00 (xfr#18, to-chk=12/31)
>f+++++++++ test/file.19
              0   0%    0.00kB/s    0:00:00            3,072 100%   23.62kB/s    0:00:00 (xfr#19, to-chk=11/31)
>f+++++++++ test/file.20
              0   0%    0.00kB/s    0:00:00            3,072 100%   23.62kB/s    0:00:00 (xfr#20, to-chk=10/31)
>f+++++++++ test/file.21
              0   0%    0.00kB/s    0:00:00            9,216 100%   69.23kB/s    0:00:00 (xfr#21, to-chk=9/31)
>f+++++++++ test/file.22
              0   0%    0.00kB/s    0:00:00            9,216 100%   60.81kB/s    0:00:00 (xfr#22, to-chk=8/31)
>f+++++++++ test/file.23
              0   0%    0.00kB/s    0:00:00            2,048 100%   13.51kB/s    0:00:00 (xfr#23, to-chk=7/31)
>f+++++++++ test/file.24
              0   0%    0.00kB/s    0:00:00            6,144 100%   40.54kB/s    0:00:00 (xfr#24, to-chk=6/31)
>f+++++++++ test/file.25
              0   0%    0.00kB/s    0:00:00           10,240 100%   56.82kB/s    0:00:00 (xfr#25, to-chk=5/31)
>f+++++++++ test/file.26
              0   0%    0.00kB/s    0:00:00           10,240 100%   56.82kB/s    0:00:00 (xfr#26, to-chk=4/31)
>f+++++++++ test/file.27
              0   0%    0.00kB/s    0:00:00            8,192 100%   41.88kB/s    0:00:00 (xfr#27, to-chk=3/31)
>f+++++++++ test/file.28
              0   0%    0.00kB/s    0:00:00            4,096 100%   20.94kB/s    0:00:00 (xfr#28, to-chk=2/31)
>f+++++++++ test/file.29
              0   0%    0.00kB/s    0:00:00            1,024 100%    5.24kB/s    0:00:00 (xfr#29, to-chk=1/31)
>f+++++++++ test/file.30
              0   0%    0.00kB/s    0:00:00            2,048 100%   10.47kB/s    0:00:00 (xfr#30, to-chk=0/31)

sent 598 bytes  received 173,856 bytes  69,781.60 bytes/sec
total size is 172,032  speedup is 0.99
