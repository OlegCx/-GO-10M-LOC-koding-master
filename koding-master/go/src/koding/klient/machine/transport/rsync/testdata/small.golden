Files: 2, Size 0
Files: 2, Size 6144
Files: 3, Size 6144
Files: 3, Size 13312
Files: 4, Size 13312
Files: 4, Size 23552
Files: 5, Size 23552
Files: 5, Size 29696
Files: 6, Size 29696
Files: 6, Size 35840
Files: 7, Size 35840
Files: 7, Size 44032
Files: 8, Size 44032
Files: 8, Size 46080
Files: 9, Size 46080
Files: 9, Size 51200
Files: 10, Size 51200
Files: 10, Size 53248
Files: 11, Size 53248
Files: 11, Size 60416
Files: 12, Size 60416
Files: 12, Size 67584
Files: 13, Size 67584
Files: 13, Size 72704
Files: 14, Size 72704
Files: 14, Size 75776
Files: 15, Size 75776
Files: 15, Size 84992
Files: 16, Size 84992
Files: 16, Size 88064
Files: 17, Size 88064
Files: 17, Size 95232
Files: 18, Size 95232
Files: 18, Size 101376
Files: 19, Size 101376
Files: 19, Size 103424
Files: 20, Size 103424
Files: 20, Size 106496
Files: 21, Size 106496
Files: 21, Size 109568
Files: 22, Size 109568
Files: 22, Size 118784
Files: 23, Size 118784
Files: 23, Size 128000
Files: 24, Size 128000
Files: 24, Size 130048
Files: 25, Size 130048
Files: 25, Size 136192
Files: 26, Size 136192
Files: 26, Size 146432
Files: 27, Size 146432
Files: 27, Size 156672
Files: 28, Size 156672
Files: 28, Size 164864
Files: 29, Size 164864
Files: 29, Size 168960
Files: 30, Size 168960
Files: 30, Size 169984
Files: 31, Size 169984
Files: 31, Size 172032
